recipes/mapseq/meta.yaml

```diff
@@ -1,16 +1,18 @@
-{% set version = "2.1.1" %}
-{% set sha256 = "b66af114f82ed9671e250995d014704b58b8e19b7ec95b1ef05a02eb8c5e8018" %}
+{% set version = "2.1.1a" %}
+{% set sha256 = "af0cb71824dd5a9b31a54b486e8444d0fb6929b4b96723f81b8a33b639aa21f7" %}
 
 package:
   name: mapseq
   version: '{{ version }}'
 
 source:
-  url: https://github.com/jfmrod/MAPseq/archive/v{{ version }}.tar.gz
+  url: https://github.com/meringlab/MAPseq/archive/v{{ version }}.tar.gz
   sha256: '{{ sha256 }}'
 
 build:
-  number: 1
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('mapseq', max_pin="2.1.1a") }}
```

Comment:
Isn't that too strict. Do you break API/ABI every release?

Reply:
hey i wasnt sure what value to put. I do not break API with every release. Do you think a value like "2" is more appropriate?

Reply:
Please read the description of this PR, section `Instructions for avoiding API, ABI, and CLI breakage issues`. Most probably you need `...='x'`

Reply:
Yes, it is `x`!
Use other recipes as examples!

Reply:
i changed it to x